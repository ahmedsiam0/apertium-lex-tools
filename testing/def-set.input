^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^whatever<n>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^bloop<n><m><sg>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^shoop<n><m><sg>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^bloop<n><f><sg>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^bloop<adj><m><pl>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^qaa<v><pret>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>/libre2<adj><sg>/libre3<adj><sg>/libre4<adj><sg>/gratis<adj><sg>$ ^qaa<adj><pl>/gloop<n>$
