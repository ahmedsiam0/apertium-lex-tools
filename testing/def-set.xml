<lrx>
	<def-seqs>
		<def-set n="bloop" tags="n.*">
			<lemma>bloop</lemma>
			<lemma>shoop</lemma>
		</def-set>
		<def-set n="blah">
			<lemma>qaa</lemma>
			<lemma>qzz</lemma>
		</def-set>
	</def-seqs>
	<rules>
		<rule>
			<match lemma="free" tags="adj.*"><select lemma="gratis"/></match>
		</rule>
		<rule weight="3">
			<match lemma="free" tags="adj.*">
				<select lemma="libre1"/>
			</match>
			<set n="bloop"/>
		</rule>
		<rule weight="4">
			<match lemma="free" tags="adj.*">
				<select lemma="libre2"/>
			</match>
			<set n="bloop" tags="n.f.*"/>
		</rule>
		<rule weight="2">
			<match lemma="free" tags="adj.*">
				<select lemma="libre3"/>
			</match>
			<set n="blah"/>
		</rule>
		<rule weight="3">
			<match lemma="free" tags="adj.*">
				<select lemma="libre4"/>
			</match>
			<set n="blah" tags="adj.*"/>
		</rule>
	</rules>
</lrx>
