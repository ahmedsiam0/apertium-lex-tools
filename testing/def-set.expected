^free<adj><sg>/gratis<adj><sg>$ ^whatever<n>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>$ ^bloop<n><m><sg>/gloop<n>$
^free<adj><sg>/libre1<adj><sg>$ ^shoop<n><m><sg>/gloop<n>$
^free<adj><sg>/libre2<adj><sg>$ ^bloop<n><f><sg>/gloop<n>$
^free<adj><sg>/gratis<adj><sg>$ ^bloop<adj><m><pl>/gloop<n>$
^free<adj><sg>/libre3<adj><sg>$ ^qaa<v><pret>/gloop<n>$
^free<adj><sg>/libre4<adj><sg>$ ^qaa<adj><pl>/gloop<n>$
